docu/files/ModuleExtensibility.html.md

```diff
@@ -49,14 +48,11 @@ Generated files of the plug-in module also contain some features related to the
  
 ### Builder module plug-in integration
 
-When the user opens a project in the Builder, all of the packages subscribed to by the project are examined, and the included modules become available in the Builder's navigation.
-At that time, all subscribed plug-in modules are combined with matching subscribed hosts, so that when the host module is shown to the user, the Builder already knows all of its currently available extensions and assigns them to their target slots. 
+When the user opens a project in the Builder, the system examines all of the subscribed packages that the project contains. Then, the included modules become available in the Builder's navigation. Next, the system combines all of the subscribed plug-in modules with matching subscribed hosts, so that when the host module displays to the user, the Builder already knows all of its currently available extensions and assigns them to their target slots. 
 
 #### Context of the plug-in module
  
-Even though a plug-in module is embedded in the host module, it doesn't share the authorization context with the host. Plug-ins have their own, completely isolated authorization context, which is not accessible by the host module (and vice versa).
-When the Builder examines the available module plug-ins, it issues an authorization request for each of them, requesting scopes as defined in the module definition.
-The authorization success redirection is captured by the plug-in (via a dedicated resource hosted by the plug-in, **/builder/auth.html**) so that it keeps the access token private. 
+Even though a plug-in module is embedded in the host module, it doesn't share the authorization context with the host. Plug-ins have their own, completely isolated authorization context, which is not accessible by the host module (and vice versa). When the Builder examines the available module plug-ins, it issues an authorization request for each of them, requesting scopes as defined in the module definition. The authorization success redirection is captured by the plug-in (via a dedicated resource hosted by the plug-in, **/builder/auth.html**) so that it keeps the access token private. 
```

Comment:
+Even though a plug-in module is embedded in the host module, it doesn't share the authorization context with the host. Plug-ins have their own, completely isolated authorization context, which is not accessible by the host module, and vice versa. When the Builder examines the available module plug-ins, it issues an authorization request for each of them, requesting scopes as defined in the module definition. The authorization success redirection is captured by the plug-in, via a dedicated resource that the plug-in **/builder/auth.html** hosts, so that it keeps the access token private.